src/SfxWeb/package.json

```diff
@@ -50,7 +50,7 @@
     "nouislider": "^15.4.0",
     "rxjs": "~6.5.4",
     "tslib": "^2.0.0",
-    "vis-data": "^6.4.1",
```

Comment:
is there a reason vis-data is removed?

Reply:
yes, it was in conflict with the vis-network library (some incompatiblity between the versions
if I can remember well). firstly, I updated the vis-data to the newest version, but the same
problem occured. then I forced the vis-network install which succeeded, but after that I got some
dependecy issues and version clashes as well. finally, I removed the vis-data and everything worked out